lib/scheduled_job.rb

```diff
@@ -78,7 +78,7 @@ def random_minutes(base, random_delta)
     end
 
     def job_exists?(job = nil)
-      conditions = ['handler like ? AND failed_at IS NULL', "%#{self.name}%"]
+      conditions = ['handler like ? AND failed_at IS NULL', "%:#{self.name}%"]
```

Comment:
Should this also have a space after? What if a job's name includes another job as a suffix, not a prefix?